File: week 0/c++/calc.cpp
#include <iostream>

int main()
{
    char op;
    double num1;
    double num2;
    double result;

    std::cout << "********** Simple Calculator **********\n";
    std::cout << "First number: ";
    std::cin >> num1;

    std::cout << "Second number: ";
    std::cin >> num2;

    std::cout << "Operator: ";
    std::cin >> op;

    result = num1;
    switch(op)
    {
        case '+':
            result += num2;
            std::cout << "result is: " << result << "\n";
            break;
        case '-':
            result -= num2;
            std::cout << "result is: " << result << "\n";
            break;
        case '*':
            result *= num2;
            std::cout << "result is: " << result << "\n";
            break;
        case '/':
            result /= num2;
            std::cout << "result is: " << result << "\n";
            break;
        default:
            std::cout << "Thats wasnt a valid operator!\n";
    }
    std::cout << "***************************************\n";
    return 0;
}

File: week 0/python/calc.py
operator = input("Enter your operator (+ - * /): ")
num1 = int(input("Enter your first number: "))
num2 = int(input("Enter your second number: "))

if operator == "+":
    print(num1 + num2)
elif operator == "-":
    print(num1 - num2)
elif operator == "*":
    print(num1 * num2)
elif operator == "/":
    print(num1 / num2)
else:
    print("Invalid operator")